### PR TITLE
Improvement: Make useless MySQL extra info messages silent & fix creating table bug

### LINKED ISSUE #5926: Feature: Make useless MySQL extra info messages silence
**Summary**

Description for this feature.

 Don't display this message if process empty data (rows = 0) 

middle_img_v2_b0f28381-aeb0-4c4b-b4ea-ef119f72b6ag

https://github.com/datafuselabs/databend/blob/5a00c5ded62467a1e9947e058d9b74b50a035fdb/query/src/servers/mysql/mysql_interactive_worker.rs#L398-L408

### PR DESCRIPTION
**Pull request by Veeupup**
Signed-off-by: Veeupup <code@tanweime.com>

I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

* Make useless MySQL extra info messages silent
* creating table with at least one column

## Changelog

- Improvement

## Related Issues

Fixes #5926 #6087

### CODE REVIEW

File: query/src/sql/parsers/parser_table.rs [hunk 1]
```diff
@@ -59,7 +59,9 @@ impl<'a> DfParser<'a> {
         }
 
         let (columns, _) = self.parse_columns()?;
-        if !columns.is_empty() && table_like.is_some() {
+        if columns.is_empty() {
```
Comment (xudong963) [inline query/src/sql/parsers/parser_table.rs:62]: How about adding a test for the case?

**Review by xudong963 — COMMENTED** [review]
Welcome back~

**Comment by xudong963** [comment]
Btw, seems you directly pushed to origin.
image

**Comment by Veeupup** [comment]
> Btw, seems you directly pushed to origin. image

ooh....  so sorry to make a stupid mistake @xudong963

**Pull request closed without merging**